net-wireless/sdrpp/metadata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+        <maintainer type="project">
+                <email>radio@gentoo.org</email>
```

Comment:
I am sorry, but NACK. You can't add some gentoo-projects as maintainer. If you wan't to add this package, this means, you will in charge of maintaining it through our proxy-maint-project. Are you willing to do that?